### PR TITLE
Code coverage improvement

### PR DESCRIPTION
**Pull request by paulodamaso**
For #81 
- Updated code coverage values on `pom.xml`
- added two tests to improve the code coverage; both tests failed, which means that the tested classes are bugged; left two puzzles to fix those classes

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#84](https://github.com/artipie/docker-adapter/issues/84) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/b5b862eb-9de3-49a4-bd5d-ba5211094798); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by paulodamaso** [comment]
@Vatavuk ping

File: src/test/java/com/artipie/docker/http/StartUploadTest.java [hunk 1]
```diff
@@ -93,4 +96,29 @@ void shouldReturnInitialUploadStatus() {
...
+     * @todo #81:30min StartUpload is not behaving correctly
+     *  StartUpload should throw an exception in case of an unexpected path,
+     *  but it isn't working. Correct the code and enable this test.
+     */
+    @Test
+    @Disabled
+    void shouldThrowExceptionOnUnexpectedPath() {
+        new Assertion<>(
+            "Did not throw exception",
+            () -> this.slice.response(
```
Comment (Vatavuk) [inline src/test/java/com/artipie/docker/http/StartUploadTest.java:109]: @paulodamaso we need to execute request, `response.send(new FakeConnection(out)).toCompletableFuture().join()`. You can observe this behavior inside `RsHasStatus`

File: src/test/java/com/artipie/docker/http/PullLayerTest.java [hunk 2]
```diff
@@ -91,6 +94,32 @@ void shouldReturnLayer() {
...
+     * @todo #81:30min PullLayer is not behaving correctly
+     *  PullLayer should throw an exception in case of an unexpected path,
+     *  but it isn't working. Correct the code and enable this test.
+     */
+    @Test
+    @Disabled
+    void shouldThrowExceptionOnUnexpectedPath() {
+        new Assertion<>(
+            "Did not throw exception",
+            () -> this.slice.response(
```
Comment (Vatavuk) [inline src/test/java/com/artipie/docker/http/PullLayerTest.java:107]: @paulodamaso we need to execute request, `response.send(new FakeConnection(out)).toCompletableFuture().join()`. You can observe this behavior inside `RsHasStatus`

File: src/test/java/com/artipie/docker/http/PullLayerTest.java [hunk 3]
```diff
@@ -91,6 +94,32 @@ void shouldReturnLayer() {
         );
     }
 
+    /*
+     * @todo #81:30min PullLayer is not behaving correctly
```
Comment (Vatavuk) [inline src/test/java/com/artipie/docker/http/PullLayerTest.java:98]: @paulodamaso Remove this puzzle after test modification.

File: src/test/java/com/artipie/docker/http/StartUploadTest.java [hunk 4]
```diff
@@ -93,4 +96,29 @@ void shouldReturnInitialUploadStatus() {
         );
     }
 
+    /*
+     * @todo #81:30min StartUpload is not behaving correctly
```
Comment (Vatavuk) [inline src/test/java/com/artipie/docker/http/StartUploadTest.java:100]: @paulodamaso Remove this puzzle after test modification.

**Comment by Vatavuk** [comment]
@paulodamaso ping

**Comment by 0crat** [comment]
@vatavuk[/z](https://www.0crat.com/u/vatavuk) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by Vatavuk** [comment]
@paulodamaso ping

**Comment by Vatavuk** [comment]
@g4s8 can we close this PR? @paulodamaso is not responding

**Comment by g4s8** [comment]
@Vatavuk ok, @paulodamaso I'm closing this, since last review comment was submitted 5 days ago and was not answered, and PR is 10 days old.

**Comment by 0crat** [comment]
Code review was too long (8 days), architects (@g4s8) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
Pull request [#84](https://github.com/artipie/docker-adapter/issues/84) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@paulodamaso please make sure all problems found by the code reviewer (@Vatavuk) were addressed by the pull request author (you, @paulodamaso )

**Comment by paulodamaso** [comment]
@g4s8 @Vatavuk Sorry, This PR became obsolete as the codebase evolved

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by paulodamaso** [comment]
> @paulodamaso please make sure all problems found by the code reviewer (@Vatavuk) were addressed by the pull request author (you, @paulodamaso )

@sereshqua I will, thanks

**Pull request closed without merging**